worker/src/lib.rs

```diff
@@ -50,7 +50,7 @@ fn verify_signature(data: &str, pk: &JWK, sig: &str) -> Result<()> {
 
 fn initialize_logging() {
     use log::Level;
-    console_log::init_with_level(Level::Error).expect("error initializing log");
+    console_log::init_with_level(Level::Trace).expect("error initializing log");
```

Comment:
```suggestion
    console_log::init_with_level(Level::Error).expect("error initializing log");
```

Reply:
several comments are marked as resolved but without resolution, is it simply because you haven't pushed everything yet?